collect-logs-win.yaml

```diff
@@ -70,6 +70,22 @@
       failed_when: False
       tags: always
 
+    - name: Add collect-event-log powershell script and fe configs
+      win_template:
+        src: "windows/{{ item }}"
+        dest: "{{ win_dir.event_log }}\\{{ item }}"
+      with_items: ["eventlogcss.txt", "eventlogjs.txt", "collect-event-log.ps1"]
+
+    - name: Execute collect-event-log script
```

Comment:
collect logs playbooks should never fail in case of errors, we want them to always run to the end not stop in case of one error or missing log file